bin/snare

```diff
@@ -148,14 +146,15 @@ if __name__ == '__main__':
     parser.add_argument("--server-header", help="set server-header", default='nginx/1.3.8')
     parser.add_argument("--no-dorks", help="disable the use of dorks", type=str_to_bool,  default=True)
     parser.add_argument("--log-dir", help="path to directory of the log file", default='/opt/snare/')
+    parser.add_argument("--path", help="path to save the page to be cloned", required=False, default='/opt/')
     args = parser.parse_args()
-    base_path = '/opt/snare/'
-    base_page_path = '/opt/snare/pages/'
-    snare_uuid = snare_setup()
+    base_path = os.path.join(args.path,'snare')
+    base_page_path = os.path.join(base_path,'pages')
```

Comment:
What if I want to use different paths fo pages and snare? I wanted to reuse cloned pages from /opt/ and I was unable to do that. Do we want this functionality?

Reply:
I have added the functionality that you're requesting